group1/requirements/views/projects.py

```diff
@@ -266,9 +267,10 @@ def get_attachments(request, projectID):
 
 @user_can_access_project()
 def upload_attachment(request, projectID):
-
     if 'file' not in request.FILES:
         raise IOError("Missing file")
+    if not re.match("^[a-zA-Z0-9_]*$", request.FILES['file'].name):
+        raise IOError("Invalid File name");
```

Comment:
This is very nitpicky, but the word "File" should probably be lower case: "file".